src/client/src/components/DMReport.jsx

```diff
@@ -64,6 +65,49 @@ const DMReport = () => {
     })
   }
 
+  const ban = async (senderID, reportId) => {
+    axios
+      .post(`http://localhost:9000/admin/ban/${senderID}`, {
+        isBan: true,
+      })
+      .then((response) => {
+        console.log(response.data);
+        console.log(reportId);
+        axios.delete(`http://localhost:9000/reports/delete`, {
```

Comment:
Would it be better to call the deleteReport function above instead of rehandling the axios call?

Reply:
changed it to call the deleteReport function